net.granjow.slowmovideo.yaml

```diff
@@ -0,0 +1,84 @@
+app-id: net.granjow.slowmovideo
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+x-add-extensions:
+    org.freedesktop.Platform.ffmpeg-full:
+        directory: lib/ffmpeg
+        version: "21.08"
+        add-ld-path: "."
+
+cleanup-commands:
+    - mkdir -p /app/lib/ffmpeg
+
+command: slowmoUI
+rename-icon: AppIcon
+rename-desktop-file: slowmoUI.desktop
+#rename-appdata-file: hugin.appdata.xml
+finish-args:
+  - --device=dri
+  - --socket=x11
+  - --socket=pulseaudio
+  # Because the file chooser somehow doesn't show the files on the host, we export some directory for reading pcap files
+  - --filesystem=xdg-public-share
```

Comment:
are you sure this is the right one?

Reply:
so I *believe* that *some* folder needs to be shared, because the app creates folders which, AFAIK, doesn't work as expected when selecting a path through the portal. I thought "public" was a good choice because users may be more okay with leaking their data of that folder to malicious apps or maintainers than their potentially more private "videos" folder.

Reply:
I agree. I put xdg-videos now.